docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -418,6 +418,9 @@ Below you will find tables describing all available types.
 
 === Basic Types
 
+.ServiceType
+Allowed values: `ClusterIP`, `NodePort`, `LoadBalancer`, `ExternalName`
```

Comment:
Is it an enum? If so it should be `cluster-ip`, `node-port`...

Reply:
Looks like they are enums - https://quarkus.io/guides/all-config#quarkus-kubernetes_quarkus.kubernetes.service-type

Reply:
Looks like we have a bug with the default value. At the very least, it should be hyphenated, or display as is if `@DefaultConverter` is used.

Reply:
It is an enum, but the values I provides should work (I tested the `NodePort`)

Reply:
But I'll update it in any case

Reply:
Fixed

Reply:
> It is an enum, but the values I provides should work (I tested the `NodePort`)

Yes, it should work because the `HyphenateEnumConverter` was made to be lenient in a way that it will default to using `valueOf()` if the hyphenated case does not work. But the idea was to push for hyphenation as the default as much as possible.

Reply:
Gocha, thanks